### PR TITLE
Add create repo template

### PR DESCRIPTION
**Pull request by fisherxu**
Add create repo template, so tsc and SIG members can approve it.

### CODE REVIEW

**Comment by fisherxu** [comment]
As discripted in tsc charter, tsc should responsible for approving the creation/deletion of GitHub repositories, I'll create another PR for repo deletion.

/cc @kubeedge/tsc @kubeedge/maintainers @WillardHu @Shelley-BaoYue  @zhiyingfang2022

**Comment by fisherxu** [comment]
cc @fujitatomoya

**Comment by WillardHu** [comment]
SGTM

**Review by chendave — COMMENTED** [review]
seems like the first three lines are not breaking correctly, see: https://github.com/kubeedge/community/blob/eb3203ada443bba3859d633e719a30cb5f6b7252/.github/ISSUE_TEMPLATE/repo-create-application.md

File: .github/ISSUE_TEMPLATE/repo-create-application.md
Comment (fujitatomoya) [inline .github/ISSUE_TEMPLATE/repo-create-application.md:14]: nit: can we just use `labels: ["xxx"]` on the header instead? related to https://github.com/kubeedge/community/issues/159. this does not block this PR to be merged.

**Review by fujitatomoya — APPROVED** [review]
lgtm with minor nit.

**Comment by JoeyHwong-gk** [comment]
We really need this but a more concise suggestion:

```markdow

### Repository Name:
<!-- Provide the proposed name of the repository -->

### Repository Description:
<!-- Provide a brief description of the repository's purpose and goals -->

### SIG or Working Group:
<!-- Specify the SIG or Working Group that will oversee this repository -->

### Repository Owners:
<!-- List the names and GitHub handles of the proposed repository owners -->

### Repository Collaborators:
<!-- who should be listed as approvers and reviewers in OWNERS -->

### Purpose and Scope:
<!-- Opinion, Describe the intended purpose and scope of the repository -->

### Roadmap:
<!-- Provide a brief overview of the project roadmap and planned milestones -->

### License:
<!-- Specify the open-source license under which the project will be released, default: Apache2 -->

### Additional Information:
<!-- Additional context for request -->


```

**Comment by fisherxu** [comment]
> seems like the first three lines are not breaking correctly, see: https://github.com/kubeedge/community/blob/eb3203ada443bba3859d633e719a30cb5f6b7252/.github/ISSUE_TEMPLATE/repo-create-application.md

Good catch, have fixed :) @chendave

**Comment by fisherxu** [comment]
@JoeyHwong-gk Thanks for the good suggestion, and I've add two items:  1. for who should have the write access, then they can manage the PRs, issues, and publish the new release 2. for who should have the admin access(subproject owners), they can manage overall the project, include collaborators, CI, etc. :)

Some clarifications: 1. `Working Group` does not own subproject, only SIG owns. 2. Roadmap will not required for a new project  3. Project License should be `Apache License, Version 2.0` as described in https://github.com/cncf/foundation/blob/main/charter.md#11-ip-policy

> We do need this template, but a slightly different idea for you.
> 
> ```
> 
> ### Repository Name:
> 
> 
> ### Repository Description:
> 
> 
> ### SIG or Working Group:
> 
> 
> ### Repository Owners:
> 
> 
> ### Repository Collaborators:
> 
> 
> ### Purpose and Scope:
> 
> 
> ### Roadmap:
> 
> 
> ### License:
> 
> 
> ### Additional Information:
> 
> ```

**Comment by fisherxu** [comment]
ping @kubeedge/tsc @kubeedge/maintainers @fujitatomoya :)

**Review by kevin-wangzefeng — COMMENTED** [review]
/lgtm
Thanks

**Pull request merged**
